var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -13,8 +13,9 @@ class Rocblas(CMakePackage):
     homepage = "https://github.com/ROCmSoftwarePlatform/rocBLAS/"
     url      = "https://github.com/ROCmSoftwarePlatform/rocBLAS/archive/rocm-3.5.0.tar.gz"
 
-    maintainers = ['haampie']
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
```

Comment:
@haampie are you okay with this? We could also have all 3 of you as maintainers on all rocm packages.

Reply:
I'm fine with this